File: Clojure/personalized_hello.clj
(defn hello []
	(println "Enter your name:")
	(let [name (read-line)] 
	(println "Hello" name))

)

(hello)

;how to get user input
;https://xvrdm.github.io/2017/06/08/clojure-for-pythonista-user-input-loop-conditional/

File: Swift/personalized_hello.swift
func greet() -> String {
    let greeting = "Hello!"
    
    print("please Enter your name")
    if let name = readLine() {
        return "hello, \(name)!"
    }
    return greeting
}

print(greet())

//get input
//https://www.programiz.com/swift-programming/basic-input-output